packages/cli/src/models/local/LocalBaseController.js

```diff
@@ -110,6 +115,57 @@ export default class LocalBaseController {
     return this.hasTypeIdentifier(contractDataPath, "t_function_baredelegatecall_nonpayable$__$returns$_t_bool_$")
   }
 
+  getUninitializedBaseContracts(contractDataPath) {
+    if (!fs.exists(contractDataPath)) return []
+    const contractJson = fs.parseJson(contractDataPath)
+    // Check whether the contract has base contracts
+    const baseContracts = contractJson.ast.nodes.find(n => n.name === contractJson.contractName).baseContracts
+    if (baseContracts.length == 0) {
+      return []
+    }
+    // Make a dict of base contracts that have "initialize" function
+    const baseContractsWithInitialize = {}
+    for (const baseContract of baseContracts) {
+      const baseContractName = baseContract.baseName.name
+      const baseContractPath = Contracts.getLocalPath(baseContractName)
+      const baseContractAbi = fs.parseJson(baseContractPath).abi
+      if (baseContractAbi.find(fn => fn.type === "function" && fn.name === "initialize")) {
```

Comment:
Could we generalize this check to also include functions that have a modifier named [`initializer`](https://github.com/zeppelinos/zos/blob/master/packages/lib/contracts/Initializable.sol#L31), even if the function name is not `initialize`?

Reply:
Yes, I'll add check for this.